### DISCUSSION TITLE
Android installation - Termux error: Segmentation fault

### DISCUSSION THREAD

**Discussion by piyushub** [quackduck/uniclip#21] [Q&A]
Hi, installed on ubuntu & macOS, working good.

On Android - Termux, Termux:API app & pkg installed. Then uniclip downloaded, unzipped, installed at $PREFIX/usr/bin/uniclip path.
when executing command: uniclip ; getting this error/message: Segmentation fault

Did I missed anything while setting up? Please help

![Screenshot from 2023-08-04 03-09-50](https://github.com/quackduck/uniclip/assets/44274066/ea057cc8-085d-474c-8c9e-ccb7104806b0)

**Comment by piyushub** [comment]
Same error if trying to run in - client and server mode. (however I will use this android device in client mode)

**Reply by quackduck** [reply]
Did you download the right build of uniclip? It could also be better to try to build from source.

**Reply by piyushub** [reply]
Hi, Thanks for quick response.

I have downloaded: uniclip_2.3.6_Linux_arm64.tar.gz ; for

Device: Samsung Galaxy A70
Processor: Qualcomm's Snapdragon 675 (6GB RAM and 128GB)
OS: Android version 11


On googling I found, generally Segmentation fault error comes - due to a process's attempts to access memory regions that it shouldn't.

Don't know how to fix it.
